CHANGELOG.rst

```diff
@@ -13,6 +13,8 @@ not released
 
 * DROPPED support for python versions < 3.8
 * UPDATED REQUIREMENT pytz is now required >= 2018.7
+* NEW test REQUIREMENT: packaging
```

Comment:
This is already required by a test dependency, so it's not strictly a "new" requirement.